### DISCUSSION TITLE
Installing compendium on Android

### DISCUSSION THREAD

**Discussion by xmbk64** [kinkofer/FightClub5eXML#200] [Q&A]
Hi,  I've read the readme but can't figure how to download and install on an android tablet. I have dropbox,  but don't know what link to click to get a file downloaded that Fight Club will recognize.  I've gone thru the process on an iPad, no problem.  Tia for any help.

**Comment by abc** [comment]
Hey @xmbk64, just to let you know, this repo is unfortunately abandoned and development has moved over to ogre-project/FightClub5eXML for the time being, at least until @kinkofer returns and adds some contributors to this repo.

The repo that you posted this question to has no way to directly download the content and it has to be compiled manually, but if you visit the new ogre-project repo, it contains pre compiled XML compendium files that you can download directly on the releases page. 

Hope this helps, any questions let me know!

**Reply by leefujin** [reply]
Do you have a link to the ogre project. I can't find it

**Reply by abc** [reply]
Yep, no problem, it's here: https://github.com/ogre-project/FightClub5eXML

**Comment by xmbk64** [comment]
Awesome, thanks!
